controller/replica_controller.go

```diff
@@ -192,6 +192,21 @@ func (rc *ReplicaController) handleErr(err error, key interface{}) {
 	rc.queue.Forget(key)
 }
 
+func (rc *ReplicaController) isOrphaned(replica *longhorn.Replica) (bool, error) {
+	node, err := rc.ds.GetNodeRO(replica.Spec.NodeID)
+
+	if err != nil && apierrors.IsNotFound(err) {
```

Comment:
There was a pattern for this, e.g. https://github.com/rancher/longhorn-manager/blob/f693bfc3fb532b2a1279b38db7b85329e2172fce/manager/setting.go#L58